tests/foreman/cli/test_host.py

```diff
@@ -1004,6 +1004,42 @@ def test_hammer_host_info_output():
     assert int(result_info['additional-info']['owner-id']) == user.id
 
 
+@pytest.mark.host_create
+@pytest.mark.tier2
+def test_positive_update_host_owner_and_verify_puppet_class_name(
+    module_env_search, module_org, module_location, module_puppet_classes, module_user
+):
+    """Update host owner and check puppet clases associated to the host
+
+    :id: 2b7dd148-914b-11eb-8a3a-98fa9b6ecd5a
+
+    :expectedresults: Host is updated with new owner
+        and has puppet class is still assigned and shown
+
+    :CaseImportance: Medium
+
+    :BZ: 1851149
```

Comment:
Include 6.8 BZ as well, which is the original one, and mark + there.
```suggestion
    :BZ: 1851149, 1809952
```

Reply:
Please add UUID to BZ QA Whiteboard

Reply:
Done :+1: